metadata_backend/conf/conf.py

```diff
@@ -171,3 +172,9 @@ def create_db_client() -> AsyncIOMotorClient:
     "publish_route": "/rpc/v2/datasets/publish_dataset",
     "catalog_pid": "urn:nbn:fi:att:data-catalog-sd",
 }
+
+metax_reference_data: Dict = {"identifier_types": {}}
+with open("metadata_backend/conf/metax_references/identifier_types.json", "r") as codes:
```

Comment:
```suggestion
with open(Path(__file__).parent / "metax_references/identifier_types.json", "r") as codes:
```
this way it will find the file in the module